docs/get_all_vcenters.md

```diff
@@ -0,0 +1,10 @@
+# get_all_vcenters
+
+Thie function will call the cluster API and get a list of all hosts within the cluster.
```

Comment:
Typo on line 3 `This function will call the cluster API and get a list of all hosts within the cluster.`